multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -163,7 +163,9 @@ write_files:
   - path: /opt/bin/kubectl
```

Comment:
`/tmp` can get garbage collected if disk space is low. I've got a commit up which makes it non-executable so it won't show up on the PATH.